src/debug.ts

```diff
@@ -108,8 +110,8 @@ export class DashmipsDebugSession extends LoggingDebugSession {
     protected async launchRequest(response: DebugProtocol.LaunchResponse, args: LaunchRequestArguments) {
         this.config = args
         this.runInTerminalRequest(...buildTerminalLaunchRequestParams(args))
-        this.client.connect(`ws://${'localhost'}:${2390}`)
-        await this.client.ready()
+        this.client.connect(args.host, args.port)
+        //await this.client.ready()
```

Comment:
I think we commented this out for debugging, but its purpose is to stall this function until the socket connects or until a timer is up (I think 1 or 10 seconds). If it no longer applies then lets just remove this (and line 126)